lib/msal-angular/package.json

```diff
@@ -36,7 +36,7 @@
     "@angular/platform-browser-dynamic": "~4.3.0",
     "@angular/router": "^4.3.0",
     "@types/handlebars": "4.0.33",
-    "@types/jasmine": "^2.8.6",
+    "@types/jasmine": "2.5.38",
```

Comment:
I think we should upgrade to @types/jasmine": "2.5.41",instead of 2.5.38 as per this github issue https://github.com/DefinitelyTyped/DefinitelyTyped/issues/14569. I have created a separate PR to fix this issue. Please revert.

Reply:
The same link has the version 2.5.38 listed as well which worked for other users. I don't think it matters which version to use as long as the doc issue is fixed. If you want , you can change the version later on after merging this PR.

Reply:
If a higher version fixes the issue we shouldn't downgrade too much. I have fixed it in a different PR and merge changes to this PR.